dev-docs/dev-config-example.yml

```diff
@@ -150,9 +150,6 @@ github:
   pulse_username: ...
```

Comment:
note for later, but this file should probably be ignored in .gitattributes and .git/CODEOWNERS?